app/models/medicaid_application.rb

```diff
@@ -28,6 +28,10 @@ def anyone_with_income?
       anyone_other_income?
   end
 
+  def nobody_married?
```

Comment:
Does this need a test? Genuine question - idk testing philosophy for units like this.

Reply:
in my mind it's small enough that it is not required and I dont think we have tests for similar methods in this class, but I am always happy to write more unit tests. do you think we should?

Reply:
Cool, sounds good to me. Just wanted to hear your thoughts. Nbd, maybe in the future if we modify this.